README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
I use cmder and `chcp 65001`. In this case it seems to be `less` which is to blame:
```
c:\data\external\R\repr (packageIQR)
λ git slog
2b62d01 Add packageIQR repr
[...]
a27c12e don<E2><80><99>t double escape latex list names

c:\data\external\R\repr (packageIQR)
λ git slog > test.txt # all is good and UTF8

c:\data\external\R\repr (packageIQR)
λ git slog |grep a27c12e
a27c12e don’t double escape latex list names

c:\data\external\R\repr (packageIQR)
λ git slog |grep a27c12e |less
a27c12e don<E2><80><99>t double escape latex list names
(END)

c:\data\external\R\repr (packageIQR)
λ git slog |grep a27c12e |more
Nicht genügend Arbeitsspeicher.
grep: write error
```

The last is the worst, because windows `more` refuses to run when `chcp 65001`...